pkg/image/prune/imagepruner.go

```diff
@@ -94,6 +96,9 @@ type PrunerOptions struct {
 	// KeepTagRevisions is the minimum number of tag revisions to preserve;
 	// revisions older than this value are candidates for pruning.
 	KeepTagRevisions int
+	// KeepOverSizeLimitImages indicates that images exceeding defined limits (openshift.io/Image)
+	// will not be considerate as candidates for pruning.
```

Comment:
I think `keep-over-sized`/`keep-over-limit` would be better, to keep them inline with other `keep-*` flags. I don't see the point in replying the word image, esp. that it already appears in `oadm prune images`. Repeating size and limit isn't valuable, either. Lastly, current name is too long. @smarterclayton @jwforres wdyt about the proposal?